### DISCUSSION TITLE
Use Head API with gatsby-plugin-react-i18next

### DISCUSSION THREAD

**Discussion by Melan1408** [gatsbyjs/gatsby#38131] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Summary

Hi,

I would like to know how to use gatsby-plugin-react-i18next with Head API. I try this but it doesn't work.

```javascript
export const Head = () =>
{
  const { t } = useTranslation();

  return (<SEO
    title={t('My title')}
  />)
}
```
I don't see any explaination in the documentation about how to use gatsby-plugin-react-i18next with Head API.

### Steps to Resolve this Issue

1. Add explaination about how to use i18n with Head API : https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/

**Comment by LekoArts** [comment]
Please see the React context note inside https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/#usage-notes
